.circleci/config.yml

```diff
@@ -12,20 +12,20 @@ defaults: &defaults
 jobs:
   test:
     docker:
-    - image: circleci/node:stretch-browsers
+      - image: circleci/node:stretch-browsers
     steps:
       - checkout
       - run:
           name: Install dependencies
           command: npm install
-      - run:
-          name: BlackDuck scan
-          command: curl -s https://detect.synopsys.com/detect.sh | bash -s -- \
-            --blackduck.url="https://qliktech.blackducksoftware.com" \
-            --blackduck.trust.cert=true \
-            --blackduck.username="svc-blackduck" \
-            --blackduck.password=${svc_blackduck} \
-            --detect.project.name="viz-bundle-qlik-multi-kpi"
+      # - run:
```

Comment:
This fails on Circle. Not sure how to fix it.

Reply:
what does it do?

Reply:
Black Duck allows you to scan applications and container images, identify all open source components, and detect any open source security vulnerabilities, compliance issues, or code-quality risks.

We removed it from client a long time ago: https://github.com/qlik-trial/sense-client/commit/02bed33397c7ff30cd7bffb734782f7d1390cdfd

I guess something that Dependabot does currently. I'll see if it is setup here

Reply:
I have enabled renovate now and also have dependabot check dependencies and create PRs for them (see PR list 😄 )

Reply:
then we might as well remove this code all together

Reply:
Yes, probably better indeed